x9.md

```diff
@@ -0,0 +1,57 @@
+# Status Account Creation and Maintenance Specification
+
+## Summary
+
+The core concept of an account in Status is a set of cryptographic keypairs.  Namely, the combination of the following:
+1. a whisper chat identity keypair
+1. a set of cryptocurrency wallet keypairs
+
+Everything else associated with the contact is either verified or derived from the above items, including:
+- Ethereum address (future verification, currently the same base keypair)
+- 3 word mnemonic name
+- identicon
+- message signatures
+
+## 1 Initial Key Generation
+### 1.1 Public/Private Keypairs 
+- An ECDSA (secp256k1 curve) public/private keypair MUST be generated via a [BIP43](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki) derived path from a [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic seed phrase.
+- The default paths are defined as such:
+    - Whisper Chat Key ($IK$): `m/43'/60'/1581'/0'/0`  (post Multiaccount integration)
+        - following [EIP1581](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1581.md)
+    - DB encryption Key ($DBK$): `m/43'/60'/1581'/1'/0` (post Multiaccount integration)
+        - following [EIP1581](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1581.md)
+    - Status Wallet paths: `m/44'/60'/0'/0'/i` starting at `i=0`
+        - following [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+        - NOTE: this (`i=0`) is also the current (and only) path for Whisper key before Multiaccount integration
+
+### 1.2 X3DH Prekey bundle creation
+- A client must create an amount of X3DH prekey bundles, each defined by the following items:
+    - Identity Key: $IK$
```

Comment:
same comment re TeX

Reply:
I would like to come together on a set of variables for each of these things as they're referred to across the specs.  Might have to make a master list of them all to see what all we're using.